assets/js/theme/account.js

```diff
@@ -96,29 +96,67 @@ export default class Account extends PageManager {
+            $bigCommerce.renderAccountPayments({
+                styles: {
+                    inputBase: {
+                        color: themeSettings['input-font-color'],
+                        borderColor: themeSettings['input-border-color'],
                     },
-                    error: {
-                        color: 'red',
+                    inputValidationError: {
+                        color: themeSettings['color-error'],
+                        borderColor: themeSettings['color-error'],
                     },
-                    placeholder: {
-                        color: '#d8d8d8',
+                    inputValidationSuccess: {
+                        color: themeSettings['color-success'],
+                        borderColor: themeSettings['color-success'],
                     },
-                    validated: {
-                        color: 'green',
+                    submitButton: {
+                        color: themeSettings['button--primary-color'],
+                        backgroundColor: themeSettings['button--primary-backgroundColor'],
+                        borderColor: themeSettings['button--primary-backgroundColor'],
+                        '&:hover': {
+                            color: themeSettings['button--primary-colorHover'],
+                            backgroundColor: themeSettings['button--primary-backgroundColorHover'],
+                            borderColor: themeSettings['button--primary-backgroundColorHover'],
+                        },
+                        '&:active': {
+                            color: themeSettings['button--primary-colorActive'],
+                            backgroundColor: themeSettings['button--primary-backgroundColorActive'],
+                            borderColor: themeSettings['button--primary-backgroundColorActive'],
+                        },
+                    },
+                    cancelButton: {
+                        color: themeSettings['button--default-color'],
+                        backgroundColor: 'transparent',
+                        borderColor: themeSettings['button--default-borderColor'],
+                        '&:hover': {
+                            color: themeSettings['button--default-colorHover'],
+                            backgroundColor: 'transparent',
+                            borderColor: themeSettings['button--default-borderColorHover'],
+                        },
+                        '&:active': {
+                            color: themeSettings['button--default-colorActive'],
+                            backgroundColor: 'transparent',
+                            borderColor: themeSettings['button--default-borderColorActive'],
+                        },
+                    },
+                    label: {
+                        color: themeSettings['form-label-font-color'],
+                    },
+                    inputWrapper: {
+
+                    },
```

Comment:
Do you still need this empty object here?

Reply:
No, we don't :) I'll remove it